src/schema/event/utils.ts

```diff
@@ -0,0 +1,31 @@
+import queryBuilder from '../../utils/queryBuilder';
+import { EventParams } from './types';
```

Comment:
You can also use generated types `import {QueryEventListArgs} from "../../types/types"`